<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Davids-MBP" failures="1" tests="8" name="api.tests.VideoServiceTest" time="5.112" errors="0" timestamp="2025-01-01T22:41:22 GMT" skipped="0">
  <testcase classname="api.tests.VideoServiceTest" name="testPostVideo" time="1.107"/>
  <testcase classname="api.tests.VideoServiceTest" name="testGetVideoById" time="0.575"/>
  <testcase classname="api.tests.VideoServiceTest" name="testPutVideo" time="0.628"/>
  <testcase classname="api.tests.VideoServiceTest" name="testDeleteVideo" time="0.581"/>
  <testcase classname="api.tests.VideoServiceTest" name="testPostVideoWithInvalidToken" time="0.555">
    <failure message="Expected 401 Unauthorized status code expected [401] but found [500]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected 401 Unauthorized status code expected [401] but found [500]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at api.tests.VideoServiceTest.testPostVideoWithInvalidToken(VideoServiceTest.java:207)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
    </failure>
  </testcase> <!-- testPostVideoWithInvalidToken -->
  <testcase classname="api.tests.VideoServiceTest" name="testPostVideoWithMissingFields" time="0.554"/>
  <testcase classname="api.tests.VideoServiceTest" name="testPostVideoWithInvalidDataFormat" time="0.539"/>
  <testcase classname="api.tests.VideoServiceTest" name="testGetNonExistentVideoById" time="0.573"/>
</testsuite> <!-- api.tests.VideoServiceTest -->
